README.md

```diff
@@ -1,12 +1,21 @@
-# Realm Shim
+# Realms Shim : OBSOLETE, INSECURE, NOT RECOMMENDED FOR USE
+
 [![Build Status][circleci-svg]][circleci-url]
 [![Coverage Status][coveralls-svg]][coveralls-url]
 [![dependency status][deps-svg]][deps-url]
 [![dev dependency status][dev-deps-svg]][dev-deps-url]
 [![License][license-image]][license-url]
 
+This repository contains a shim implementation of the [Realm API Proposal](https://github.com/tc39/proposal-realms/#ecmascript-spec-proposal-for-realms-api). 
```

Comment:
This looks good. The only suggestion I could come up with might be to add the word "legacy" in front of shim implementation, which might further drive home the point that this is outdated. (With warnings, I tend to be a bit more repetitive with language so there's no room for error.)